tests/kafka/main.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2017 Heptio Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
```

Comment:
To expedite the speed of local testing, running kafka on docker worked nicely but there are issues with connectivity from the host machine to the container. Docker compose allowed me to connect to kafka with a linked container and it is also self-documenting.